### DISCUSSION TITLE
Add `Callable` in method `get` / ` set` to handle if catch miss.

### DISCUSSION THREAD

**Discussion by smling** [xetorthio/jedis#3325] [Ideas]
**As** developer, 

**I need to** make a convenient way to assign value when cache missing when get / set in redis.

**So that** I don't need to handle cache missing when `get` and need to set it to redis again.

PoC repository: https://github.com/smling/redis-extension

**Comment by sazzad16** [comment]
@smling Thank you for your interest in Jedis and willingness to improve it.

This idea does not align with the _near future plans_ for Jedis. But let's see if this idea makes other users interested as well.
